docs/restapi.rst

```diff
@@ -284,7 +288,47 @@ This route will return all categories and their translations.
 translations
     Translated fields are stored inside a translations object. They can have any size, depending on if there is a translation. If a required language is not found, you should fall back to English.
 
+.. _api-token:
+
+Get API Token
+~~~~~~~~~~~~~
+This route will return the API token for the authenticated user. If no token
+exists, one will be generated.
+
+* **Url**: POST /api/v1/token
+
+* **Authentication**: Basic, Session
```

Comment:
Should SessionAuthentication be mentioned in the docs? It might not be very useful there but it is the truth.